src/renderer/lib/notification.ts

```
@@ -25,18 +25,13 @@
~
 export const notifyProgress = <T>(
   promise: Promise<T & { result?: boolean; error?: unknown }>,
   {
-    loadingMessage,
     successMessage,
     errorMessage,
-  }: { loadingMessage: string | Component; successMessage: string; errorMessage: string },
+  }: { successMessage: string; errorMessage: string },
```

ESLint

[error] Replace `⏎····successMessage,⏎····errorMessage,⏎·` with `·successMessage,·errorMessage`

(prettier/prettier)